src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -258,25 +215,21 @@ public IIcon getIcon( int direction, int metadata )
 		return this.getRendererInstance().getTexture( ForgeDirection.getOrientation( direction ) );
 	}
 
+	public ICustomCollision getCustomCollision( World w, int x, int y, int z )
```

Comment:
why `public` also it returns `null` outside of the own object, might be dangerous (potential new NPEs)